### DISCUSSION TITLE
A modder made fsr 2 possible in any dlss 2 game

### DISCUSSION THREAD

**Discussion by vassystem** [Blinue/Magpie#397] [Ideas]
Interesting development: [https://www.nexusmods.com/cyberpunk2077/mods/3001](url)

Thought I would share here because the consensus here was this was never going to be possible. Well turns out it was possible! Pretty cool.

**Comment by MindSerene** [comment]
对,就是这个吧
github.com/PotatoOfDoom/CyberFSR2

**Comment by GordonFreeeman** [comment]
I just came here to post literally the same link. I've been trying to get this mod to work with Hitman 3, but to no avail. Still, if it's as "easy" (😉) as hijacking the DLSS dll with FSR 2.0, it might be doable on a general scale. That's absolutely brilliant!
